libs/shared/data-access/work-api/src/lib/models/dashboard/dashboard-count.model.ts

```diff
@@ -0,0 +1,9 @@
+export class DashboardCountModel {
```

Comment:
We have a patter for handling the models already

```suggestion
export class DashboardCountModel implements SchemaModel<interface> {
```

Reply:
Thank you David, the SchemaModel interface has been implemented.